modules/doctor/dbconsistency.go

```diff
@@ -201,10 +201,10 @@ func checkDBConsistency(ctx context.Context, logger log.Logger, autofix bool) er
 			"oauth2_authorization_code", "oauth2_grant", "oauth2_authorization_code.grant_id=oauth2_grant.id"),
 		// find stopwatches without existing user
 		genericOrphanCheck("Orphaned Stopwatches without existing User",
-			"stopwatch", "user", "stopwatch.user_id=user.id"),
+			"stopwatch", "user", "stopwatch.user_id=`user`.id"),
 		// find stopwatches without existing issue
 		genericOrphanCheck("Orphaned Stopwatches without existing Issue",
-			"stopwatch", "issue", "stopwatch.issue_id=issue.id"),
+			"stopwatch", "issue", "stopwatch.issue_id=`issue`.id"),
```

Comment:
Does issue need this as well?

Reply:
No harm 🤷🏽(Simple ctrl-f shows multiple references and I have no idea when to use them 😄) I don't have a Postgres setup to test it.

Reply:
it's not consistent before and wont be after ... - I dont mid if this is changed too